### PR TITLE
Add totals row to the Daily Dropoffs Report

### PR DESCRIPTION
**Pull request by zachmargolis**
[Request in Slack](https://gsa-tts.slack.com/archives/C5E7EJWF7/p1634767439004600?thread_ts=1634652839.003400&cid=C5E7EJWF7)

see new footer row

Screen Shot 2021-10-20 at 3 34 14 PM

### CODE REVIEW

File: src/css/style.scss [hunk 1]
```diff
@@ -11,6 +11,11 @@
   text-align: right;
 }
 
+.usa-table tfoot td {
+  font-weight: bold;
+  background-color: #dedede;
+}
```
Comment (zachmargolis) [inline src/css/style.scss:17]: As far as I can tell... USWDS does not style `tfoot` at all... I might consider opening a PR that just copies the `thead` ones? 🤷

File: src/components/table.tsx [hunk 2]
```diff
@@ -5,15 +5,34 @@ export type TableRow = (string | number | VNode)[];
...
 
+type NumberFormatter = (n: number) => string;
+
 interface TableProps {
   data: TableData;
-  numberFormatter?: (n: number) => string;
+  numberFormatter?: NumberFormatter;
+}
+
+function renderRow(row: TableRow, numberFormatter: NumberFormatter) {
```
Comment (aduth) [inline src/components/table.tsx:18]: Maybe it can be a proper component `TableRow` ?

File: src/css/style.scss [hunk 3]
```diff
@@ -11,6 +11,11 @@
   text-align: right;
 }
 
+.usa-table tfoot td {
+  font-weight: bold;
+  background-color: #dedede;
```
Comment (aduth) [inline src/css/style.scss:16]: Could avoid hard-coding and use the token instead:

```suggestion
  background-color: color('base-lighter');
```

Via: https://design.login.gov/utilities/color/

File: src/components/daily-dropoffs-report.tsx [hunk 4]
```diff
@@ -51,6 +51,8 @@ function tabulate({
       "white" as unknown as number,
     ]);
 
+  const totals = funnelSteps(funnelMode).map(() => 0);
```
Comment (aduth) [inline src/components/daily-dropoffs-report.tsx:54]: Good use case for `Array#fill`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill

File: src/components/table.tsx (hunk 2 above)
Comment (zachmargolis) [reply src/components/table.tsx]: updated in  a185371dc8eb06e4b8af37f46dabb2a9f0f0de90

**Pull request merged**